edge/cmd/edgecore/app/server.go

```diff
@@ -176,7 +176,7 @@ func environmentCheck() error {
 	for _, process := range processes {
 		processName, err := process.Name()
 		if err != nil {
-			return err
+			continue
```

Comment:
For the `environmentCheck`, can we also put it into files for windows and others?

Reply:
Here we use `continue` to skip check for some processes which failed to get name. I think it can be kept in Linux as well, but we need add a comment here.